projects/effective-proposal-framework-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/epf/entity/PrisonManager.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.epf.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.annotations.Where
+import org.springframework.data.annotation.Immutable
+
+@Entity
+@Table(name = "prison_offender_manager")
+@Where(clause = "soft_deleted = 0 and active = 1")
```

Comment:
Should this be active_flag?

```suggestion
@Where(clause = "soft_deleted = 0 and active_flag = 1")
```